sources/packages/api/src/route-controllers/student/student.aest.controller.ts

```diff
@@ -177,4 +178,23 @@ export class StudentAESTController extends BaseController {
       sendFileUploadNotification,
     );
   }
+
+  /**
+   * Get the student information that represents the profile.
+   * @param studentId student id to retrieve the data.
+   * @returns student profile details.
+   */
+  @Get(":studentId")
+  @ApiNotFoundResponse({ description: "Not able to find the student." })
+  async getStudentProfile(
+    @Param("studentId") studentId: number,
+  ): Promise<StudentProfileAPIOutDTO> {
+    const student = await this.studentControllerService.getStudentProfile(
+      studentId,
+    );
+    if (!student) {
+      throw new NotFoundException("Not able to find the student.");
```

Comment:
"Student profile not found" sounds more like it.

Reply:
It seems to me that both are valid ways, is it a suggestion?

Reply:
Yes